component/org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls/pom.xml

```diff
@@ -0,0 +1,152 @@
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls</artifactId>
+    <packaging>bundle</packaging>
+    <name>Mutual TLS client authenticator</name>
+    <url>http://wso2.org</url>
+
+    <parent>
+        <groupId>org.wso2.carbon.extension.identity.oauth.addons</groupId>
+        <artifactId>identity-oauth2-extenstions</artifactId>
+        <version>1.0.3-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
+            <artifactId>org.wso2.carbon.identity.oauth</artifactId>
+            <version>5.6.29</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.core</artifactId>
+            <version>5.11.16</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven.plugins.version}</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls.internal
+                        </Private-Package>
+                        <Import-Package>
+                            org.osgi.service.*,
+                            org.osgi.framework.*,
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls.internal,
+                            org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls.utils.*,
+                            org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls.*,
+                        </Export-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Do we need dynamic imports ? Any particular reason ? Can we just have import packages instead ?